sisyphus/task.py

```diff
@@ -481,5 +481,8 @@ def get_worker_call(self, task_id=None):
         call += [gs.CMD_WORKER, os.path.relpath(self.path()), self.name()]
         if task_id is not None:
             call.append(str(task_id))
-        call = gs.worker_wrapper(getattr(self, "_job", None), self.name(), call)
+        if hasattr(self, "_job"):
+            call = self._job._sis_worker_wrapper(self._job, self.name(), call)
+        else:
```

Comment:
Is the Task not having the job assigned yet a valid situation?

Reply:
>> Is the Task not having the job assigned yet a valid situation?
>
> but having this check doesn't do any harm and it keeps the old behavior.

that was exactly my train of thought

>  I would propose to add a warning here

we can do that